scripts/nb_link.js

```diff
@@ -0,0 +1,46 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2022, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+const { execute } = require("./script_utils.js");
+const path = require("path");
+
+const nb_path = path.resolve(
+    path.join(
+        __dirname,
+        "..",
+        "python",
+        "perspective",
+        "perspective",
+        "nbextension",
+        "static"
+    )
+);
+const json_path = path.resolve(
+    path.join(
+        __dirname,
+        "..",
+        "python",
+        "perspective",
+        "perspective",
+        "extension",
+        "finos-perspective-nbextension.json"
+    )
+);
+
+(async function () {
+    try {
+        execute`mkdir -p ~/.jupyter/nbextensions/@finos/`;
+        execute`mkdir -p ~/.jupyter/nbconfig/notebook.d/`;
+        execute`ln -s ${nb_path} ~/.jupyter/nbextensions/@finos/perspective-jupyterlab`;
```

Comment:
this is a Tim special, I symlink everything in `jupyter --paths` to `~/.jupyter`, but this is not the default installation behavior. We should note this somewhere in the contributor docs

Reply:
I have a pretty plain setup and `~/.jupyter` is listed in my `jupyter --paths` (after `share/jupyter` in my venv root)… so linking this into `~/.jupyter` should make it available to jupyter, I think? I haven't actually tested the nbextension yet